docs/workflow/ci/failure-analysis.md

```diff
@@ -121,8 +121,17 @@ To unconditionally bypass the build analysis check (turn it green), you can add
 /ba-g <reason>
 ```
 
+The reasons are captured by telemetry and regularly analyzed to allow us to find most impactful opportunities for improvement. Use descriptive wording that highlights the issue. Avoid using non-specific justifications like "unrelated issues".
+
 The `Build Analysis` requests are sent to a queue. In certain scenarios, this queue can have many items to process and it can take a while for the status to be updated. If you do not see the status getting updated, be patient and wait at least 10 minutes before investigating further.
 
+While most failures can be matched via known issues, a few failures modes cannot be matched currently and it is valid to suppress them manually. Please use the following reasons to suppress them:
+
+- `/ba-g doc changes only` - build analysis won't turn green for PRs that contain documentation .md file changes only.
+- `/ba-g deadletter` - Helix infrastructure failed with "This is a helix work item crash with status: DeadLetter." error message. Validate that the coverage provided by the dead-lettered leg is not relevant to the PR first. Rerun the leg instead if the coverage is relevant.
+- `/ba-g insufficient info in logs` - There is no good unique pattern in the logs to open a known issue.
+- `/ba-g recently fixed known issue` - The fix for the known issue had already been merged, but the CI run was triggered beforehand.
```

Comment:
During telemetry analysis, it was helpful when known issue numbers were included. So, let's include issue numbers like 
`/ba-g recently fixed known issue #<Known Issue number 1>, #<Known Issue number 2>, ...`.

And, there is a similar but separate case where a fix is not ready but all related Known Issue(s) are filed. So, it'd be good to add:
`ba-g known issue #<Known Issue number 1>, #<Known Issue number 2>, ...`.